extensions/spotify/init.lua

```diff
@@ -125,4 +153,54 @@ function spotify.getCurrentTrack()
     return tell('name of the current track')
 end
 
+--- hs.spotify.getPlayerState()
```

Comment:
I think this might be better as `hs.spotify.getPlaybackState()` in case we want to add other state related things in the future. `Player` seems a bit too generic.